src/boot/ksdk1.1.0/gpio_pins.h

```diff
@@ -8,31 +8,31 @@
 /*

```

Comment:
This file contains unnecessary diffs that do not make any actual change.

Reply:
I think I've made it worse. now there are diffs everywhere :(